pkg/config/polling_manager.go

```diff
@@ -215,6 +219,18 @@ func (cm *PollingProjectConfigManager) GetConfig() (ProjectConfig, error) {
 	return cm.projectConfig, nil
 }
 
+// GetOptimizelyConfig returns the optimizely project config
+func (cm *PollingProjectConfigManager) GetOptimizelyConfig() *OptimizelyConfig {
+	cm.configLock.RLock()
+	defer cm.configLock.RUnlock()
+	if cm.optimizelyConfig != nil {
+		return cm.optimizelyConfig
+	}
+	optimizelyConfig := NewOptimizelyConfig(cm.projectConfig)
```

Comment:
This is interesting - triggering the first OptConfig only when called by client. 
Tradeoff between memory saving for most clients who do not use OptConfig vs extra delay for 1st build.
It looks good as is, and we can consider alternate setting based on feedbacks.